irohad/ametsuchi/impl/flat_file/flat_file.cpp

```diff
@@ -166,3 +126,39 @@ FlatFile::FlatFile(Identifier current_id,
   log_ = logger::log("FlatFile");
   current_id_.store(current_id);
 }
+
+nonstd::optional<Identifier> FlatFile::check_consistency(
+    const std::string &dump_dir) {
+  auto log = logger::log("FLAT_FILE");
+
+  if (dump_dir.empty()) {
+    log->error("check_consistency({}), not directory", dump_dir);
+    return nonstd::nullopt;
+  }
+
+  auto const files = [&dump_dir] {
+    std::vector<boost::filesystem::path> ps;
+    std::copy(boost::filesystem::directory_iterator{dump_dir},
+              boost::filesystem::directory_iterator{},
+              std::back_inserter(ps));
+    std::sort(ps.begin(),
```

Comment:
Fix this statement as well plz: `std::sort(ps.begin(), ps.end(), std::less<boost::filesystem::path>());`

Reply:
This seems out scope for this pull request, maybe create another task for it?